### ISSUE TITLE
Brushless Motor (IDFGH-11263)

### ISSUE THREAD

**Issue by franz-ms-muc** [espressif/esp-idf#12420]
### Answers checklist.

- [X] I have read the documentation [ESP-IDF Programming Guide](https://docs.espressif.com/projects/esp-idf/en/latest/) and the issue is not addressed there.
- [X] I have updated my IDF branch (master or release) to the latest version and checked that the issue is present there.
- [X] I have searched the issue tracker for a similar issue and not found a similar issue.

### General issue report

https://github.com/espressif/esp-idf/blob/8fc8f3f47997aadba21facabc66004c1d22de181/examples/peripherals/mcpwm/mcpwm_bldc_hall_control/main/mcpwm_bldc_hall_control_example_main.c

this Example can run 2 times ? 

for 1 ESP32 controlling 2 BL DC Motors ?

**Comment by suda-morris** [comment]
yes, esp32 has two MCPWM controllers. One controller can be used to control one BLDC (3 pair of PWMs).
